src/channel.rs

```diff
@@ -592,6 +592,11 @@ struct ChannelInner {
     channel: *mut grpc_channel,
 }
 
+// SAFETY: `grpc_channel` is safe to send between threads, and `Environment` is already `Send`.
+unsafe impl Send for ChannelInner {}
```

Comment:
It's intended to impl for `Channel` instead. Because only the APIs exposed by `Channel` are `Sync` and `Send`.

Reply:
`ChannelInner` is not public so this doesn't affect the public API, and is cleaner than having a non-`Sync` type in an `Arc`.

Reply:
It's about maintenance. For example, putting `ChannelInner` in `Arc` can be unsafe until developer takes care of the wrappers. Marking it sync and send gives the wrong suggestion to developers who may not simply put it into another wrapper without thinking twice.